Sources/LiveViewNativeCharts/Modifiers/View Fundamentals Modifiers/ChartBackgroundModifier.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ChartBackgroundModifier.swift
+//
+//
+//  Created by murtza on 22/06/2023.
+//
+
+import Charts
+import SwiftUI
+import LiveViewNative
+
+/// Adds a background to a view that contains a chart.
+///
+/// ```html
+/// <VStack template={:content} modifiers={chart_background(@native, alignment: :center)}>
+///   <chart_background:background>
+///     <Color name="system-green" />
+///   </chart_background:background>
+///   <Chart>Chart Content!</Chart>
+/// </VStack>
+/// ```
```

Comment:
Typically this modifier would be used on the `<Chart>` element directly, could the example be updated to show it like that instead of with a `VStack`?